lib/auth/vault.js

```diff
@@ -28,118 +26,81 @@ function vaultSignatureCb(err, userInfo, log, callback) {
     return callback(null, new AuthInfo(userInfo.message.body));
 }
 
-const vault = {};
-
 /**
  * authenticateV2Request
  *
- * @param {string} accessKey - user's accessKey
- * @param {string} signatureFromRequest - signature sent with request
- * @param {string} stringToSign - string to sign built per AWS rules
- * @param {string} algo - either SHA256 or SHA1
-* @param {RequestContext} requestContext - an instance of a RequestContext
+ * @param {string} params - the authentication parameters as returned by
```

Comment:
I may be wrong about it, but this entire file is not used right now, this changes need to be done against s3 :)